README.md

```diff
@@ -18,52 +18,87 @@ Coalton can be written in files:
 ```lisp
 (in-package #:coalton-user)
 
-(coalton-toplevel
-  (define-type Symbol
-    (Symbol String))
-
-  (define (symbol-name sym)
-    (match sym
-      ((Symbol s) s)))
+(named-readtables:in-readtable coalton:coalton)
 
+(coalton-toplevel
+  ;; Define Coalton `Symbol`s as Lisp `cl:symbol`s.
+  (repr :native cl:symbol)
+  (define-type Symbol)
+
+  ;; Bind a Lisp function into Coalton.
+  (declare sym (String -> Symbol))
+  (define (sym s)
+    "Create a new symbol named `s`."
+    (lisp Symbol (s)
+      (cl:intern s)))
```

Comment:
Could this be something like `(cl:intern s :keyword)`. A bare `inern` in example code makes me nervous.

Reply:
yeah that sounds reasonable; done